homework2_count_vow.py

```diff
@@ -0,0 +1,73 @@
+# Task description:

```

Comment:
Do not forget about docstring smth aka
"""
Hometask02.

Task description...
"""
There you can place task description and own comments. So use docstring as file caption instead of comment lines.